fusor-ember-cli/app/mirage/factories/deployment.js

```diff
@@ -31,8 +31,8 @@ export default Mirage.Factory.extend({
   custom_preprend_name: null,
   enable_access_insights: true,
   openstack_undercloud_password: "dummy password",
-  upstream_consumer_uuid: "7ffddefd-aacb-4192-a999-01beb7c2e473",
-  upstream_consumer_name: "tsanders-rhci",
+  upstream_consumer_uuid: null,
+  upstream_consumer_name: null,
```

Comment:
I nullified these values for the mock data. Otherwise, I don't get to the credentials route. Is this the right behavior, @jmrodri?